Dockerfile

```diff
@@ -10,6 +10,9 @@ RUN cd freebindfree && make
 # Typescript build
 FROM node:18 as builder
 
+RUN apt-get update && apt-get install -y node-gyp --no-install-recommends
```

Comment:
It seems node:18.17 works node:18.18 is missing node-gyp for some reason.

Reply:
We've had this before where minor image versions break (and then because we target a major version, you end up not noticing).

The change looks good then. I think in the future we're just going to have to start pinning versions strictly as we can't be having this.